src/default_theme/section._

```diff
@@ -16,15 +16,14 @@
   <% } %>
 
   <%= md(section.description) %>
-
-  <div class='pre p1 fill-light mt0'><%= signature(section) %></div>
+ 
   <% if (section.type) { %>
     <p>
       Type:
       <%= formatType(section.type) %>
     </p>
   <% } %>
-  <% if (section.augments.length) { %>
+  <% if (section.arguments && section.augments.length) { %>
```

Comment:
Looks like a typo - arguments should be augments.

Reply:
I would be glad to fix the typo, sorry about that.